pagopa/io-app
# feat: [#176479978] Add react native bundle visualizer as dev dependency

## Short description
This pr adds react native bundle visualizer in order to measure the size of the bundle.

![immagine](https://user-images.githubusercontent.com/26501317/104463903-9f4b8b80-55b2-11eb-8298-c1769b6fb9e3.png)


## List of changes proposed in this pull request
- Added `"react-native-bundle-visualizer": "^2.2.1"` ad `devDependency`

## How to test
- Run `yarn run react-native-bundle-visualizer` in order to see the size distribution
